javaagent/build.gradle.kts

```diff
@@ -94,7 +94,7 @@ dependencies {
   baseJavaagentLibs(project(":instrumentation:opentelemetry-instrumentation-annotations-1.16:javaagent"))
   baseJavaagentLibs(project(":instrumentation:executors:javaagent"))
   baseJavaagentLibs(project(":instrumentation:internal:internal-application-logger:javaagent"))
-  baseJavaagentLibs(project(":instrumentation:internal:internal-class-loader:javaagent"))
+  baseJavaagentLibs(project(":instrumentation:internal:internal-class-loader:javaagent", configuration = "shaded"))
```

Comment:
do you think we could make this `shaded` configuration override the default one so we don't have to propagate similar changes wherever this module is used ? This could also remove the need to selectively exclude the default configuration in the change below.

Reply:
I don't know how to do that. It could be beneficial in other places too. If you know how this could be done then that would be awesome.